bpy_addon_build/build_context/hook_definitions.py

```diff
@@ -19,24 +24,44 @@ def build_action_prebuild(ctx: BuildContext, action: str, console: Console) -> N
     if ctx.config.build_actions is None and len(ctx.cli.actions):
         print("Actions must be defined to use them!")
         return
-    if action in ctx.cli.actions:
-        if action not in ctx.api.action_mods:
-            print("Action not in API!")
-            return
-        if hasattr(ctx.api.action_mods[action], "prebuild"):
-            res: Optional[Union[BpyError, BpyWarning]] = ctx.api.action_mods[
-                action
-            ].pre_build()
-
-            if res is not None:
-                if isinstance(res, BpyError):
-                    console.print(f"{res.msg}", style="red")
-                    quit(-1)
-                elif isinstance(res, BpyWarning):
-                    console.print(f"{res.msg}", style="yellow")
-
-
-def build_action_main(ctx: BuildContext, action: str, console: Console) -> None:
+    if action not in ctx.cli.actions:
+        return
+    if action not in ctx.api.action_mods:
+        print("Action not in API!")
```

Comment:
I'm also a bit confused while trying to test this. I'm basically _always_ getting the action is not found, because the "action" is just matching the passed in variable name, as opposed to the hook name.

```
if action not in ctx.api.action_mods:
        print("Action pre_install not in API!", action)
        return
```

The actual printout line I get is `Action pre_install not in API! fast`

The line I'm compiling with is `bab -b fast -v 4.0` where `fast` corresponds to a `fast.py` file. What I was expecting is that it would call the `pre_install` hook

Maybe I'll understand better if we already did the implementation of the fast command, ie the ignore pattern for MCprep_resources paired with pre_install moving the installed MCprep_resources folder to a temp place, and then putting it back in the post_install step. That's all I'm hoping to accomplish here!

My full fast.py snippet:

```python
from bpy_addon_build.api import BabContext

print("Did this run...? fast")  # this seems to never print


def main(ctx: BabContext) -> None:
    print("Dev build, adding mcprep_dev.txt...")
    with open("mcprep_dev.txt", "w") as f:
        f.write("This file is used to enable debugging mode in MCprep, you can ignore this")


def pre_install(ctx: BabContext) -> None:
    print("HELLO")
    print(ctx)


def post_install(ctx: BabContext) -> None:
    print("POST INSTALL")
    print(ctx)
```

Reply:
Pushed some debug information for the API. What does the output look like when passing `-dbg`

Reply:
Welp, TL;DR I forgot we also have to add in this line, so my fast arg wasn't being recognized at all. It all seems working well for me now

```
  fast:
    script: "action-scripts/fast.py"
```

Let me just do a little more of a test before approving this